src/components/Toggle/Toggle.stories.tsx

```diff
@@ -2,6 +2,8 @@ import React, { useState } from 'react';
 import Toggle from '@/components/Toggle';
 import { Meta } from '@storybook/react';
 import mdx from './Toggle.stories.mdx';
+import classNames from 'classnames';
+import styles from './ToggleTest.css';
```

Comment:
@branquinhoaa could you please rename this file to Toggle.stories.css? This is to maintain consistency in the project architecture.